app/assets/stylesheets/email.css.scss

```
@@ -177,6 +177,10 @@ h6 {
   @include u-border(1px);
 }
 
+.border-top-width-0 {
+  border-top: 0;
+}
```

The other utilities in this file follow from USWDS. AFAIK, `border-top-width-0` doesn't exist in USWDS. [But `border-top-0` does](https://designsystem.digital.gov/utilities/border/#:~:text=24px-,Border%20on%20the%20top,-.border%2Dtop).

```suggestion
.border-top-0 {
  @include u-border-top(0);
}
```

---

[PR# 10839](https://github.com/18F/identity-idp/pull/10839) handles this comment.